modules/openapi-generator/src/main/resources/swift5/libraries/urlsession/URLSessionImplementations.mustache

```diff
@@ -126,6 +124,7 @@ private var urlSessionStore = SynchronizedDictionary<String, URLSession>()
         }
 
         let cleanupRequest = {
+            urlSessionStore[urlSessionId]?.finishTasksAndInvalidate()
```

Comment:
Interesting, I didn't know this API.
Can you please comment on the need of this?
Just curious.
Thanks 😊

Reply:
This invalidates the session after the all tasks of this session have completely finished. New tasks cannot be started on this session.
This call is not necessary, but good practice, since we're creating a session for each request, so after a task as finished we don't need the session anymore.

Reply:
Nice, thanks 👍